### ISSUE TITLE
Odd "gap" in Saved themes, even w/ pagination

### ISSUE THREAD

**Issue by pdehaan** [mozilla/Themer#194]
TL;DR: Should we increase the saved themes per page from 8 to 12?

### Steps to reproduce:
1. Create some theme, or choose a preset.
2. Spam the [Save] button in "Save your theme" (enough times that you get pagination, so 10-20x).
3. Scroll down to the "Saved themes" heading.

### Actual results:
![firefox_themer_-_saved_themes](https://user-images.githubusercontent.com/557895/37724717-a1d1d6f6-2cee-11e8-955d-9aa05a679221.png)

I see pagination, but an odd gap in the lower-right corner. You'll only see the gap at certain viewports, where you have 3 themes horizontally. Browser too wide, you get 4 columns. Too narrow, you get 2 columns. It's a real Goldilocks problem.

This happens because we seemingly display 8x saved themes per page. Ideally we'd have a number divisible by 2, 3, and 4. So in order to avoid gaps, we should increase the number of saved themes per page from 8 to 12, so it'd be 2 columns x 6 rows, 3 columns by 4 rows, or 4 columns of 3 rows.

**Comment by pdehaan** [comment]
I reckon it's just bumping this pref: https://github.com/mozilla/Themer/blob/413aa25b78865392779fdb680735d52d32568d03/src/web/lib/components/SavedThemeSelector/index.js#L26

**Comment by Shruti9520** [comment]
Hi @pdehaan ,
Is this issue is about just changing perPage?

**Comment by pdehaan** [comment]
Currently just a question on whether we _should_.

**Comment by johngruen** [comment]
Yes, we *should*

@Shruti9520 feel free to grab this!

**Comment by Shruti9520** [comment]
@pdehaan @johngruen Taking this :-)
